IQKeyBoardManager/Categories/IQUIView+Hierarchy.m

```diff
@@ -161,17 +161,24 @@ -(BOOL)_IQcanBecomeFirstResponder
     return _IQcanBecomeFirstResponder;
 }
 
-- (NSArray*)responderSiblings
+- (NSArray *)responderSiblings
 {
-    //	Getting all siblings
+    // Getting all siblings
     NSArray *siblings = self.superview.subviews;
     
     //Array of (UITextField/UITextView's).
     NSMutableArray *tempTextFields = [[NSMutableArray alloc] init];
     
-    for (UITextField *textField in siblings)
-        if ([textField _IQcanBecomeFirstResponder])
+    for (UITextField *textField in siblings) {
+        if (textField.subviews.count) {
+            [textField.subviews enumerateObjectsUsingBlock:^(UIView *subview, NSUInteger idx, BOOL *stop) {
```

Comment:
This piece of code is not acceptable, because responderSiblings method should return only the view pure siblings. This method should not return all children siblings too at any case.